Fhi.HelseId.Web/Middleware/ProtectedPaths.cs

```diff
@@ -33,16 +32,14 @@ public ProtectPaths(RequestDelegate next, ProtectPathsOptions options,ILogger<Pr
             _accessDeniedPath = options.AccessDeniedPath;
         }
 
-        public async Task Invoke(HttpContext httpContext,
-            IAuthorizationService authorizationService)
+        public async Task Invoke(HttpContext httpContext, IAuthorizationService authorizationService)
         {
             var path = httpContext.Request.Path;
             logger.LogTrace($"ProtectedPaths: Checking path: {path}");
             if (!_excludedPaths.Any(path.StartsWithSegments))
             {
-                if (!httpContext.User.Identity.IsAuthenticated)
+                if (httpContext.User.Identity == null || !httpContext.User.Identity.IsAuthenticated)
```

Comment:
```suggestion
                if ( !httpContext.User.Identity?.IsAuthenticated ?? true)
```

Reply:
This suggestion was a bigger brainteaser for me. Shorter yes, more readable ? @martinmine @yne-bv

Reply:
To remove all the null-checks it can be done like this
`if (httpContext.User.Identity is not { IsAuthenticated: true })`

Reply:
@yne-bv I like, pushed update.